### PR TITLE
Generate CDN SDK for 2020-09-01 AFDX support

### PR DESCRIPTION
**Pull request by ChenglongLiu**
Regenerate CDN SDK for 2020-09-01 AFDX support https://github.com/Azure/azure-rest-api-specs/tree/7a71c15c380d7b9061496698f3679a19aa63191d/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01

### CODE REVIEW

**Comment by ChenglongLiu** [comment]
This request for an SDK for our upcoming AFDX feature.

**Review by nisha-bhatia — CHANGES_REQUESTED** [review]

**Comment by nisha-bhatia** [comment]
@ChenglongLiu What feature is this code being re-generated for? Are new properties being added? Thank you.

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs [hunk 1]
```diff
@@ -59,8 +59,8 @@ public CdnWebApplicationFirewallPolicy()
...
-            : base(location, id, name, type, tags)
+        public CdnWebApplicationFirewallPolicy(string location, Sku sku, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), SystemData systemData = default(SystemData), PolicySettings policySettings = default(PolicySettings), RateLimitRuleList rateLimitRules = default(RateLimitRuleList), CustomRuleList customRules = default(CustomRuleList), ManagedRuleSetList managedRules = default(ManagedRuleSetList), IList<CdnEndpoint> endpointLinks = default(IList<CdnEndpoint>), string provisioningState = default(string), string resourceState = default(string), string etag = default(string))
+            : base(location, id, name, type, tags, systemData)
```
[the remark this answers is not in the document]
Comment (ChenglongLiu) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs:63]: These codes are automatically generated from swagger define.

[the remark this answers is not in the document]
Comment (nisha-bhatia) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs:63]: @ChenglongLiu Could you please provide link to the new Swagger?

**Comment by ChenglongLiu** [comment]
> @ChenglongLiu What feature is this code being re-generated for? Are new properties being added? Thank you.

@nisha-bhatia  AFDX is a new service and will public review on 02/18. Serval resources are added in AFDX. We need to release the new version SDK first for service sign off.

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChenglongLiu) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs:63]: readme: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cdn/resource-manager
version 2020-09-01: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01

[the remark this answers is not in the document]
Comment (ChenglongLiu) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs:63]: @nisha-bhatia

**Comment by jorinmejia** [comment]
@nisha-bhatia, can we now merge this now that all the checks have passed? Our development is currently blocked on this PR.

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (nisha-bhatia) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/Models/CdnWebApplicationFirewallPolicy.cs:63]: @ChenglongLiu Can you please point me to the Swagger PR that added the new SystemData property to CdnWebApplicationFirewallPolicy?

**Review by nisha-bhatia — CHANGES_REQUESTED** [review]

**Comment by jorinmejia** [comment]
@nisha-bhatia, the SystemData property was added in the below PR. Its an ARM property and not specific to our service. 

https://github.com/Azure/azure-rest-api-specs/pull/11655/files#diff-81f6c7aad1a5b5fcf16bd0e7ddeb95be5e384f50d43e3081c31b4e7c9ed8b856

**Comment by nisha-bhatia** [comment]
@jorinmejia Could you please provide the Swagger link where the SystemData property is introduced for this method? I don't see it in the linked Swagger PR.

**Comment by ChenglongLiu** [comment]
> @jorinmejia Could you please provide the Swagger link where the SystemData property is introduced for this method? I don't see it in the linked Swagger PR.

@nisha You can find the SystemData at line 5452 of cdn.json 
https://github.com/Azure/azure-rest-api-specs/pull/11655/files#diff-81f6c7aad1a5b5fcf16bd0e7ddeb95be5e384f50d43e3081c31b4e7c9ed8b856R5452

It's read only system data
```
 "SystemData": {
      "description": "Read only system data",
      "type": "object",
      "properties": {
        "createdBy": {
          "type": "string",
          "description": "An identifier for the identity that created the resource"
        },
        "createdByType": {
          "$ref": "#/definitions/IdentityType",
          "description": "The type of identity that created the resource"
        },
        "createdAt": {
          "type": "string",
          "format": "date-time",
          "description": "The timestamp of resource creation (UTC)"
        },
        "lastModifiedBy": {
          "type": "string",
          "description": "An identifier for the identity that last modified the resource"
        },
        "lastModifiedByType": {
          "$ref": "#/definitions/IdentityType",
          "description": "The type of identity that last modified the resource"
        },
        "lastModifiedAt": {
          "type": "string",
          "format": "date-time",
          "description": "The timestamp of resource last modification (UTC)"
        }
}
```

**Comment by jorinmejia** [comment]
@nisha-bhatia, this SystemData property is also a requirement by ARM for RPs. We had to include it in our swagger to pass validations and other tests when merging into the Azure REST Specs repo.

**Review by nisha-bhatia — APPROVED** [review]

**Pull request merged**